sympy/parsing/latex/__init__.py

```diff
@@ -31,11 +42,19 @@ def parse_latex(s, strict=False):
     (sqrt(a) + 1)/b
     >>> expr.evalf(4, subs=dict(a=5, b=2))
     1.618
+    >>> func = parse_latex(r"\int_1^x \dfrac{\mathrm{d}t}{t}", backend="Lark")
+    >>> func.evalf(subs={x: 2})
```

Comment:
I guess this example can be removed, so that ANTLR remains the only dependency for this doctest. This is how it is done in the [`sample` function](https://github.com/sympy/sympy/blob/395e820b114d2b169483354f1f4ee2f439faa292/sympy/stats/rv.py#L1063-L1160) in [`stats/rv.py`](https://github.com/sympy/sympy/blob/395e820b114d2b169483354f1f4ee2f439faa292/sympy/stats/rv.py#L1063-L1160), which also offers multiple backends.